log/log.go

```diff
@@ -25,6 +25,16 @@ func (l Logger) With(keysvals ...interface{}) Logger {
 }
 
 /*
+WrapError can wrap error with kerrs to structured log
+*/
+func (l Logger) WrapError(err error) log.Logger {
+	if err == nil {
+		return l
+	}
+	return l.WithError(kerrs.Wrapv(err, ""))
```

Comment:
Why do you need to pass an empty string here?

Reply:
Wrapv takes a message parameter. what should we pass?